install.sh

```diff
@@ -170,6 +170,9 @@ wget -N $URL/lib/package_helpers.rb
 rm -f /usr/local/bin/cc
 ln -s /usr/local/bin/gcc /usr/local/bin/cc
 
+# install docopt
+gem install docopt
```

Comment:
@yancouto is correct.  We should include the library instead of installing via gem.  From the docopt project page:
```
Docopt is available via rubygems:

gem install docopt
Alternatively, you can just drop lib/docopt.rb file into your project--it is self-contained.
```
Dropping the lib file makes more sense.